store/registry.go

```diff
@@ -232,13 +260,23 @@ func (rs *privateRegistryStore) GetAgentsIfChanged(scanner string) ([]*config.Ag
 	var agentConfigs []*config.AgentConfig
 
 	// load by image references
-	for i, agentImage := range rs.cfg.LocalModeConfig.BotImages {
-		if len(agentImage) == 0 {
+	for i, bot := range rs.cfg.LocalModeConfig.BotImages {
+		if len(bot.Image) == 0 {
 			continue
 		}
 		// forta-agent-1, forta-agent-2, forta-agent-3, ...
 		agentID := strconv.Itoa(i + 1)
-		agentConfigs = append(agentConfigs, rs.makePrivateModeAgentConfig(agentID, agentImage))
+
+		for i := 0; i < int(bot.Redundancy); i++ {
```

Comment:
same here (thinking shards, not redundancy; redundancy is only interpreted by assigner)